sdk/src/types/api/core/response.rs

```diff
@@ -262,11 +262,11 @@ pub struct ValidatorResponse {
 #[serde(rename_all = "camelCase")]
 pub struct IssuanceBlockHeaderResponse {
     /// Blocks that are strongly directly approved.
-    pub strong_parents: StrongParents,
+    pub strong_parents: basic::StrongParents,
     /// Blocks that are weakly directly approved.
-    pub weak_parents: WeakParents,
+    pub weak_parents: basic::WeakParents,
     /// Blocks that are directly referenced to adjust opinion.
-    pub shallow_like_parents: ShallowLikeParents,
+    pub shallow_like_parents: basic::ShallowLikeParents,
```

Comment:
I don't know about these. That would mean we can't build a ValidationBlock from the API return. I don't think the number returned by the API should be validated at all. It's only when we try to build a Block with these that we should validated.

Reply:
hmm, okay, I'll give it a try